asserts/asserts.go

```diff
@@ -37,6 +37,17 @@ const (
 	noAuthority typeFlags = iota + 1
 )
 
+// MetaHeaders is a list of headers in assertions which are about the assertion
+// itself.
+var MetaHeaders = []string{
```

Comment:
I'd've made this a `[...]string` instead of a `[]string` but maybe I'm being awkward